Add "How to contact Marketing" (#3725)
sourcegraph/about: handbook/marketing/index.md

- [Sign up for Marketing office hours](https://docs.google.com/spreadsheets/d/1o8rHntP0j6-JFryhc0_g6MKEI6l3dZE7NTKasyQx46M/edit#gid=0)
- Join the [Marketing team meeting](https://calendar.google.com/event?action=TEMPLATE&tmeid=dWozMjcxcjlnazMxZ2xjcnNkOHZrdjBwNXZfMjAyMTA3MTJUMTUzMDAwWiByZWJlY2NhQHNvdXJjZWdyYXBoLmNvbQ&tmsrc=rebecca%40sourcegraph.com&scp=ALL) (every second week)
- Slack channel: #marketing

### Who to go to for what

The table below breaks down the capabilities of each team within Marketing. Once you've identified which team is relevant to your query, check out their handbook page or contact them via their Slack channel or contact form. If you're still not sure which team you should contact, please submit a [general request](#General-Marketing-requests-and-feedback) or reach out in #marketing.

| Team                | What we can help with | Slack channels | Submit an idea                                                                            |
|---------------------|-----------------------|----------------|-------------------------------------------------------------------------------------------|

> | [Brand](brand/index.md)               |                       |                |                                                                                           |

Author: @fabicastp Could you please fill out the `What we can help with | Slack channels | Submit an idea` fields for Brand & Creative? Tables in markdown are fiddly so if you haven't encountered them before you can just tell me here what to add if you prefer 😃

Reviewer: Hi Rebecca! Thank you for doing this. I'll write them here because I'd break that table in a second. 

Branding, graphic design, website/digital experiences, photo/video production, swag, social media, and event planning.

Author: Thanks @fabicastp! Can you also share which Slack channels people should reach out to you in, and how to submit an idea (I presume through the Asana form but want to make sure)?

> | Communications      |                       |                |                                                                                           |

Author: @amieroth Could you please fill out the `What we can help with | Slack channels | Submit an idea` fields for Communications? Tables in markdown are fiddly so if you haven't encountered them before you can just tell me here what to add if you prefer 😃

Author: @amieroth I put a placeholder in for your row so I can merge this in the meantime! I'm happy to add your bits whenever you're ready 😁

> | Community Relations          |                       |                |                                                                                           |

Author: @samswag Could you please fill out the `What we can help with | Slack channels | Submit an idea` fields for Community Relations (or nominate someone from your team to do it)?

Reviewer 2: @rebeccadee  Done

Author: Thank you @unicodeveloper! Not everyone at Sourcegraph has access to Asana. How can people submit an idea if they aren't using it? Should we send them to the general marketing request form?

Reviewer 2: Yes please @rebeccadee

> | [Content Marketing](content/index.md)   | Writing, editing, and strategic support for content that raises awareness and builds trust with our developer audience. Content channels and formats we own: the blog (except for release posts, which are managed by Product Marketing), web articles, guides, quizzes.                       | #content-chat  | [Content Proposal form](https://form.asana.com/?k=gU7tdLoh3TyyrSa-w0-pCQ&d=7195383522959) |

Author: @content-hacker @jennybergen Input welcome here please, I am sure we can communicate this better 😆

Reviewer 3: @rebeccadee can you add reports to the list of formats? And should the podcast be mentioned here as well?

Author: ```suggestion
| [Content Marketing](content/index.md)   | Writing, editing, and strategic support for content that raises awareness and builds trust with our developer audience. Content channels and formats we own: the blog (except for release posts, which are managed by Product Marketing), reports, web articles, guides, quizzes.                       | #content-chat  | [Content Proposal form](https://form.asana.com/?k=gU7tdLoh3TyyrSa-w0-pCQ&d=7195383522959) |
```

> | [Demand Gen](demandgen.md)          |                       |                |                                                                                           |

Author: @sarah-reece Could you please fill out the `What we can help with | Slack channels | Submit an idea` fields for Demand Gen (or nominate someone from your team to do it)? As above, if you're not familiar with tables in Markdown you can just leave a comment for me to add it 😁

> | [Developer Education](education.md) |                       |                |                                                                                           |

Reviewer 4: ```suggestion
| [Developer Education](education.md) |                       | #developer-education | [Dev Ed Content Board](https://app.asana.com/0/1199987786100094/board)                                                                                           |
```

> | [Developer Education](education.md) |                       | #developer-education | [Dev Ed Content Board](https://app.asana.com/0/1199987786100094/board)                                                                                           |

Author: @ltagliaferri Marek already filled out some of your fields (thanks Marek!) but could you please add the `What we can help with` field? Thanks!

> | [Product Marketing](product_marketing.md)   |                       |                |                                                                                           |

Author: @abschu Could you please fill out the `What we can help with | Slack channels | Submit an idea` fields for PMM (or nominate someone from your team to do it)? As above, if you're not familiar with tables in Markdown you can just leave a comment for me to add it 😁

Reviewer 5: @rebeccadee - What we can help with: Product launches, product messaging and positioning, pricing and packaging, customer stories, Customer Advisory Board, and analyst relations.

Slack channels #customer-advisory-board, #pricing, #release-post 

Submit an idea: https://form.asana.com/?k=hNaq42PNshdQ1FjjEUKjLA&d=7195383522959

Thank you!

- Join the [Marketing team meeting](https://calendar.google.com/event?action=TEMPLATE&tmeid=dWozMjcxcjlnazMxZ2xjcnNkOHZrdjBwNXZfMjAyMTA3MTJUMTUzMDAwWiByZWJlY2NhQHNvdXJjZWdyYXBoLmNvbQ&tmsrc=rebecca%40sourcegraph.com&scp=ALL) (every second week)
- Slack channel: #marketing

### Who to go to for what

The table below breaks down the capabilities of each team within Marketing. Once you've identified which team is relevant to your query, check out their handbook page or contact them via their Slack channel or contact form. If you're still not sure which team you should contact, please submit a [general request](#General-Marketing-requests-and-feedback) or reach out in #marketing.

| Team                | What we can help with | Slack channels | Submit an idea                                                                            |
|---------------------|-----------------------|----------------|-------------------------------------------------------------------------------------------|
| [Brand](brand/index.md)               |                       |                |                                                                                           |

> | [Social Media](brand/index.md)               |  Shaping social strategy under the Brand Team, Social @ Sourcegraph is responsible for the stewardship of the Sourcegraph brand social channels, and enablement of team member social profiles. We are accountable for the organic editorial calendar and partner with other departments across the organization to provide organic social promotion.                       |  #social-media-action & #social-media-posts              |  Send requests, ideas, and industry updates to the #social-media-action channel                                                                                         |

Author: @kristen-sundberg Thank you so much for adding your details here ✨ Thinking of this from the perspective of someone outside marketing coming to this page to find out who can assist them with an ask, maybe we could word this more explicitly? The column is "What we can help with" so the answer could be "Promoting new content and events via Sourcegraph branded social channels, guidance on personal social media profiles and engagement, ...." I'll leave it to you, but hopefully you get the idea!

Reviewer: Hi Rebecca! I'm adding my info here again. It's not letting me reply to your comment anymore. This is for Brand: 

Slack channels: #brand-creative #swag
Creative requests: https://form.asana.com?k=Zv_DWGDitZCYr0FXY3bvHg&d=7195383522959
Swag requests:https://form.asana.com?k=ysBKSzSA2xjRifqqEvFbJQ&d=7195383522959

Thank you again for doing this <3

| Communications      |                       |                |                                                                                           |

> | Community Relations   |  Speaking, attending, organizing meetups, virtual events, hackathons, conferences. OSS community outreach. Creating engaging technical content that assists developers in their daily work and adoption of Sourcegraph, in form of articles, blog posts, demos, cookbooks and videos.  We own the Developer Tool Time web series & the Sourcegraph Champions program. | #community-relations |       https://app.asana.com/0/1200326741766875/board                                                                                    |

Reviewer 2: @samswag

> | Community Relations   |  Speaking, attending, organizing meetups, virtual events, hackathons, and conferences. OSS community outreach. Creating engaging technical content that assists developers in their daily work and adoption of Sourcegraph, in form of articles, blog posts, demos, cookbooks and videos.  We own the Developer Tool Time web series & the Sourcegraph Champions program. | #community-relations |       https://app.asana.com/0/1200326741766875/board                                                                                    |

Author: ```suggestion
| Community Relations   |  Speaking, attending, organizing meetups, virtual events, hackathons, and conferences. OSS community outreach. Creating engaging technical content that assists developers in their daily work and adoption of Sourcegraph, in form of articles, blog posts, demos, cookbooks and videos.  We own the Developer Tool Time web series & the Sourcegraph Champions program. | #community-relations |       [Marketing Request form](https://form.asana.com/?k=hNaq42PNshdQ1FjjEUKjLA&d=7195383522959)                                                                                     |
```

| [Content Marketing](content/index.md)   | Writing, editing, and strategic support for content that raises awareness and builds trust with our developer audience. Content channels and formats we own: the blog (except for release posts, which are managed by Product Marketing), reports, web articles, guides, quizzes.                       | #content-chat  | [Content Proposal form](https://form.asana.com/?k=gU7tdLoh3TyyrSa-w0-pCQ&d=7195383522959) |
| [Demand Gen](demandgen.md)          |                       |                |                                                                                           |
| [Developer Education](education.md) | Educational resources for developers | [#developer-education](https://app.slack.com/client/T02FSM7DL/C026GJE9DDX/user_groups/S023NTWTTEX), [#sourcegraph-learn](https://app.slack.com/client/T02FSM7DL/C02173Y7JGH/user_groups/S023NTWTTEX) | [File an issue on Sourcegraph Learn](https://github.com/sourcegraph/learn/issues/new)                                                                                          |

> | [Product Marketing](product_marketing.md)   |      Product launches, product messaging and positioning, pricing and packaging, customer stories, Customer Advisory Board, and analyst relations.                 |           #customer-advisory-board, #pricing, #release-post     |              [Marketing Request form]             (https://form.asana.com/?k=hNaq42PNshdQ1FjjEUKjLA&d=7195383522959)                                                                |

Author: ```suggestion
| [Product Marketing](product_marketing.md)   |      Product launches, product messaging and positioning, pricing and packaging, customer stories, Customer Advisory Board, and analyst relations.                 |           #customer-advisory-board, #pricing, #release-post     |              [Marketing Request form](https://form.asana.com/?k=hNaq42PNshdQ1FjjEUKjLA&d=7195383522959)                                                                |
```

    - [Naming Process for Products, Features and Programs](brand/naming_process_for_products_features_and_programs.md)
    - [Working with Us](brand/production_process.md)

- [Marketing roles - open positions](https://sourcegraph.com/careers)

> ## Members

Reviewer 3: @rebeccadee is there a magical way to have this section auto-update when we update it on the "Marketing Team" page? Or do we need to always update in two places?

Author: @jennybergen I'm not sure I understand your question – this _is_ the marketing team page?
